pkg/rescaffold/migrate.go

```diff
@@ -100,8 +100,24 @@ func getOutputPath(currentWorkingDirectory, outputPath string) (string, error) {
 	return "", err
 }
 
-func kubebuilderInit(_ store.Store) error {
+func kubebuilderInit(store store.Store) error {
 	var args []string
 	args = append(args, "init")
+	args = append(args, getInitArgs(store)...)
 	return util.RunCmd("kubebuilder init", "kubebuilder", args...)
 }
+
+func getInitArgs(store store.Store) []string {
+	var args []string
+	plugins := store.Config().GetPluginChain()
+	if len(plugins) > 0 {
+		args = append(args, "--plugins")
+		args = append(args, plugins...)
```

Comment:
No, I tried this and it will not compile: 
```
too many arguments in call to append
        have ([]string, string, []string)
        want ([]string, ...string)
make: *** [Makefile:57: build] Error 1
```